tests/test_return_value_policy_override.cpp

```diff
@@ -51,9 +53,13 @@ const obj &return_const_reference() {
     return value;
 }
 
-std::shared_ptr<obj> return_shared_pointer() { return std::make_shared<obj>("shared_pointer"); }
+std::shared_ptr<obj> return_shared_pointer() {
+    return std::shared_ptr<obj>(new obj("shared_pointer"));
```

Comment:
Should be `std::make_shared`, right?

Reply:
Initially I used `std::make_shared` and `std::make_unqiue`, but then I see errors `note: ‘std::make_unique’ is only available from C++14 onwards`, so I modified both of them. I should be able to use `std::make_shared` with C++11. I will update the PR.